src/pages/Equipments.vue

```diff
@@ -0,0 +1,307 @@
+                />
+                <EquipmentsIcon 
+                  v-bind:equipmentType="EquipmentType.Nothing"
+                  v-bind:equipmentId="'nothing'"
+                />
+              </div>
+              <div class="flex-row">
+                <EquipmentsIcon 
+                  v-bind:equipmentType="EquipmentType.Nothing"
+                  v-bind:equipmentId="'nothing'"
+                />
+                <EquipmentsIcon 
+                  v-bind:equipmentType="EquipmentType.PremiumRing"
+                  v-bind:equipmentId="'premiumring'"
+                  @clicked="equipmentSelected"
+                />
+              </div>
+              <div class="flex-row">
+                <EquipmentsIcon 
+                  v-bind:equipmentType="EquipmentType.Nothing"
+                  v-bind:equipmentId="'nothing'"
+                />
+                <EquipmentsIcon 
+                  v-bind:equipmentType="EquipmentType.Nothing"
+                  v-bind:equipmentId="'nothing'"
+                />
+              </div>
+            </div>
+          </div>
+        </div>
+        <div>
+          <EquipmentsSelector 
+            v-bind:selectedType="currentSelectedType"
+            v-bind:selectedId="currentSelectedId"
+          />
+        </div>
+      </div>
+      <div v-if="curEquipmentStats">
+        <span class="stat-line">Stats for the current equipment : </span>
+        <span class="stat-line">Weapon Power : {{ curEquipmentStats.weaponPower }}</span>
+        <span class="stat-line">Speed : {{ curEquipmentStats.speed }}</span>
+        <span class="stat-line">Defense : {{ curEquipmentStats.defense }}</span>
+        <span class="stat-line">Strength : {{ curEquipmentStats.strength }}</span>
+        <span class="stat-line">Agility : {{ curEquipmentStats.agility }}</span>
+        <span class="stat-line">Wisdom : {{ curEquipmentStats.wisdom }}</span>
+        <span class="stat-line">Luck : {{ curEquipmentStats.luck }}</span>
+        <span class="stat-line">Reach : {{ curEquipmentStats.reach }}</span>
+
+        <span class="stat-line" 
+          v-for="(misc, i) in curEquipmentStats.miscellaneous"
+          :key="i"
+        >
+          {{ misc.value }} {{ misc.line }}
+        </span>
+
+      </div>
+    </div> 
+</template>
+
+<script lang="ts">
+import { defineComponent, ref, watch } from "vue";
+
+import EquipmentsIcon from "~/components/equipments/EquipmentsIcon.vue"
+import EquipmentsSelector from "~/components/equipments/EquipmentsSelector.vue";
+
+import { useCharacters } from "~/composables/Characters";
+import { Equipment, EquipmentType, EquipmentUtilities } from "~/composables/Equipments";
+
+export default defineComponent({
+  name: "Equipments",
+  components: {
+    EquipmentsIcon,
+    EquipmentsSelector
+  },
+  
+  setup() {
+    const { curCharacter } = useCharacters();
+
+    let curEquipmentStats = EquipmentUtilities.getCurrentSlotStats();
+
+    let currentTab = ref(<number>0);
```

Comment:
(nit): I think `ref<Number>(0)` is a better definition, since it binds the type of the ref value.